docs/assets/i18n/en/design-system.json

```diff
@@ -102,6 +102,11 @@
   "notifications-wall-reply": "{firstUser} <b>replied</b> to {secondUser}'s message <br><br> {postTitle}",
   "notifications-wall-reply-own-message": "{user} <b>replied</b> to your message <br><br> {postTitle}",
   "notifications-wall-reply-multiple-users": "{firstUser} and {number} other users <b>replied</b> to {secondUser}'s wall <br><br> {postTitle}",
+	"notifications-article-comment-followed-article-comment": "{user} <b>commented</b> on the article: {articleTitle}",
+	"notifications-article-comment-reply-own-comment": "{user} <b>replied</b> to your comment on the article: {articleTitle}",
+	"notifications-article-comment-reply-followed-comment": "{user} <b>replied</b> to a comment you are following on the article: {articleTitle}",
+	"notifications-article-comment-comment-mention": "{user} <b>mentioned</b> you in a comment on the article: {articleTitle}",
+	"notifications-article-comment-reply-mention": "{user} <b>mentioned</b> you in a reply on the article: {articleTitle}",
```

Comment:
It looks like GH has issues with updating whitespace changes. These are spaces, I swear.